src/common/localization/en-us.ts

```diff
@@ -506,5 +506,66 @@ export const english: IAppStrings = {
                 "An error occurred while predicting regions in the current asset. \
                 Please verify your active learning configuration and try again"
         }
+    },
+    wasteTypes: {
```

Comment:
@SchnaebeleMarc  In VoTT translations are in typescript file. Do we need to do it in json?

Reply:
Yes it's better because the tool we use has a large c hoice of file format but not ts files